### ISSUE TITLE
Pandas 0.23.4 TypeError: Already tz-aware

### ISSUE THREAD

**Issue by fl4p** [influxdata/influxdb-python#671]
The UTC[ tz localization](https://github.com/influxdata/influxdb-python/blob/master/influxdb/_dataframe_client.py#L205) raises a `TypeError: Already tz-aware, use tz_convert to convert.` with pandas 0.23.4 . 

I think this a suitable fix:
```
            if df.index.tzinfo is None:
                df.index = df.index.tz_localize('UTC')
```

**Comment by grainednoise** [comment]
I ran into the same problem. The proposed fix solves the problem for me.

**Comment by nathanielatom** [comment]
Any ETA on when this fix will be merged and released?

**Comment by lovasoa** [comment]
I got the same problem. Is something preventing the PR from being merged ?

<details>

```
File "/lib/python2.7/site-packages/influxdb/_dataframe_client.py", line 185, in query
return self._to_dataframe(results, dropna)
File "/lib/python2.7/site-packages/influxdb/_dataframe_client.py", line 205, in _to_dataframe
df.index = df.index.tz_localize('UTC')
File "/lib/python2.7/site-packages/pandas/core/accessor.py", line 91, in f
return self._delegate_method(name, *args, **kwargs)
File "/lib/python2.7/site-packages/pandas/core/indexes/datetimelike.py", line 721, in _delegate_method
result = operator.methodcaller(name, *args, **kwargs)(self._data)
File "/lib/python2.7/site-packages/pandas/core/arrays/datetimes.py", line 1049, in tz_localize
raise TypeError("Already tz-aware, use tz_convert to convert.")
TypeError: Already tz-aware, use tz_convert to convert.
```

</details>

**Comment by AlnsV** [comment]
This solves the problem, happening to me just now...

**Comment by freshwuzhere** [comment]
Me too :-(

**Comment by lovasoa** [comment]
#679 fixes this. @xginn8  should [merge it soon](https://github.com/influxdata/influxdb-python/pull/679#issuecomment-467851424)

**Comment by clslgrnc** [comment]
Should be solved in 5.2.2
